src/components/Home.jsx

```diff
@@ -1,28 +1,18 @@
-import React from "react";
-import {Navigation} from "./index";
+import React from 'react';
+import { Footer, Navigation } from './index';
+import './home.css';
 
-function Home() {
+function Home(props) {
   return (
-    <div className="home">
-      <Navigation />
-      <div className="container">
-        <div className="row align-items-center my-5">
-          <div className="col-lg-7">
-            <img
-              className="img-fluid rounded mb-4 mb-lg-0"
-              src="http://placehold.it/900x400"
-              alt=""
-            />
-          </div>
-          <div className="col-lg-5">
-            <h1 className="font-weight-light">Home</h1>
-            <p>
-              This is the home page.
-            </p>
-          </div>
+    <>
+      <Navigation {...props} />
+      <div className='home'>
+        <div className='col-lg-5'>
+          <h1 className='font-weight-light'>Welcome!</h1>
```

Comment:
i think we can remove this line, it's not being used bc it's blocked by the picture

Reply:
yeah I know, that line can be removed in a future merge. same with the picture not lining up, the styling isn't perfect it's just a rough start but it's fine for this merge

Reply:
fixed